website/docs/docs/collaborate/dbt-explorer-faqs.md

```diff
@@ -0,0 +1,176 @@
+
+</expandable>
+
+<expandable alt_header="Where do recommendations come from? Can they be customized?" >
+
+Recommendations largely mirror the best practice rules from the `dbt_project_evaluator` package. At this time, recommendations can’t be customized. In the future, dbt Explorer will likely support recommendation customization capabilities (for example, in project code). 
+
+</expandable>
+
+## Column-level lineage
+
+<expandable alt_header="What are the best use cases for column-level lineage in dbt Explorer?" >
+
+Column-level lineage in dbt Explorer can be used to improve many data development workflows, including:
+
+- **Audit** &mdash; Visualize how data moves through and is used in your dbt project
+- **Root cause** &mdash; Improve time to detect and resolve data quality issues, tracking back to the source
+- **Impact analysis** &mdash; Trace transformations and usage to avoid introducing issues for consumers
+- **Efficiency** &mdash; Prune unnecessary columns to reduce costs and data team overhead
+
+</expandable>
+
+<expandable alt_header="Does the column-level lineage remain functional even if column names vary between models?" >
+
+Yes. Column-level lineage can handle name changes across instances of the column in the dbt project. 
+
+</expandable>
+
+<expandable alt_header="Can multiple projects leverage the same column definition?" >
+
+No. Cross-project column lineage is supported in the sense of viewing how a public model is used across projects, but not on a column-level. 
+
+</expandable>
+
+<expandable alt_header="Is column-level lineage data available through the API?" >
+
+Yes. Column-level lineage is available through a beta endpoint in the Discovery API to dbt Cloud Enterprise plan customers. 
+
+</expandable>
+
+<expandable alt_header="Can column descriptions be propagated down in downstream lineage automatically?" >
+
+Not currently, but this type of functionality is planned in future releases.
+
+</expandable>
+
+<expandable alt_header="Is column-level lineage also available in the development tab?" >
+
+Not currently, but we plan to incorporate column-level awareness across features in dbt Cloud in the future.
+
+</expandable>
+
+
+
+## Availability, access, and permissions
+
+<expandable alt_header="How can non-developers interact with dbt Explorer?" >
+
+Read-only users can consume metadata in dbt Explorer. More bespoke experiences and exploration avenues for analysts and less-technical contributors will be provided in the future. 
+
+</expandable>
+
+<expandable alt_header="Does dbt Explorer require a specific dbt Cloud plan?" >
+
+dbt Explorer is available on the dbt Cloud Team or Enterprise plans. Certain features within dbt Explorer, like multi-project lineage and column-level lineage, are only available on the Enterprise plan. 
+
+</expandable>
+
+<expandable alt_header="Will dbt Core users be able to leverage any of these new dbt Explorer features?" >
```

Comment:
This was already asked and answered above

Reply:
ah, i see what you mean (the on-prem question). good catch! the question is asked slightly in a different way and is also under a diff category. i'm getting the signal that this is for a diff "persona" . i'll leave this as is for now.